src/helix/subscriptions/get_broadcaster_subscriptions_events.rs

```diff
@@ -0,0 +1,235 @@
+//! Returns a list of subscription events.
+//! [`get-broadcaster-subscriptions-events`](https://dev.twitch.tv/docs/api/reference#get-broadcaster-subscriptions-events)
```

Comment:
link doesn't work because of https://twitch.uservoice.com/forums/310213-developers/suggestions/39847468-get-broadcaster-subscriptions-events , should probably not be a link